src/main/java/jdbchomework/console/ProjectsMenu.java

```diff
@@ -0,0 +1,67 @@
+package jdbchomework.console;
+
+import jdbchomework.controller.ProjectController;
+import org.slf4j.LoggerFactory;
+
+public class ProjectsMenu extends AbstractMenu {
+
+    private static final String PROJECT_NOT_FOUND = "Project not found";
+    private static final String PLEASE_ENTER_PROJECT_NAME = "Please enter project name: ";
+    private static final String PLEASE_ENTER_PROJECT_ID = "Please enter project ID: ";
+    private static final String PLEASE_ENTER_PROJECT_COST = "Please enter project cost: ";
+
+    private ProjectController projectController;
+
+    public ProjectsMenu(MenuContents menuContents, ProjectController projectController) {
+        super(LoggerFactory.getLogger(ProjectsMenu.class), menuContents);
+        this.menuContents = menuContents;
+        this.projectController = projectController;
+    }
+
+    public void menu() {
+        mark:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)